stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClientImpl.kt

```diff
@@ -428,7 +428,7 @@ internal class ChatClientImpl(
     }
 
     override fun getVersion(): String {
-        return BuildConfig.VERSION_NAME + "-" + BuildConfig.BUILD_TYPE
+        return BuildConfig.STREAM_CHAT_CLIENT_VERSION + "-" + BuildConfig.BUILD_TYPE
```

Comment:
Library projects no longer generate `VERSION_CODE` and `VERSION_NAME` values in the `BuildConfig` ([see AGP 4.1 release notes](https://developer.android.com/studio/releases/gradle-plugin#version_properties_removed_from_buildconfig_class_in_library_projects)). These values were often incorrect, and overwritten by manifest merges.

The solution for programatically accessing the values is to add them to the `BuildConfig` for ourselves (see the changes in the `build.gradle` files).